SETUP.md

```
@@ -0,0 +1,396 @@
+   - In VS Code, right-click `index.html`
+   - Click "Open with Live Server"
+   - Browser opens automatically at http://localhost:5500
+
+---
+
+### Step 5: Test the Application
+
+#### Test Backend API
+
+Open browser and visit:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)